lib/loader/tests/assembly/index.ts

```diff
@@ -1,3 +1,4 @@
+/// <reference path="../../../../std/assembly/index.d.ts" />
```

Comment:
Suggesting to add a `tsconfig.json` to the directory instead

Reply:
👍